libcxx/include/__utility/pair.h

```diff
@@ -74,6 +75,31 @@ struct _LIBCPP_TEMPLATE_VIS pair
   _LIBCPP_HIDE_FROM_ABI pair(pair const&) = default;
   _LIBCPP_HIDE_FROM_ABI pair(pair&&)      = default;
 
+  // Make pair trivially copyable if we have a way to do it
```

Comment:
This comment is somewhat misleading since we make it trivially copyable if we have a way to do it *and* were asked to do it in the ABI configuration. We could instead say

```
// When we are requested for pair to be trivially copyable by the ABI macro, we use defaulted members
// if it is both legal to do it (i.e. no references) and we have a way to actually implement it, which requires
// the __enable_if__ attribute before C++20.
```